resources/conf.edn

```diff
@@ -0,0 +1,10 @@
+{:cassandra-session {:hosts #profile {:development ["localhost"]
+                                      :production ["node-0.cassandra.mesos"
+                                                   "node-1.cassandra.mesos"
+                                                   "node-2.cassandra.mesos"]}
```

Comment:
all the hosts we want it to access, yes. I imagine that it doesn't ahve to be ALL of them if that becomes a problem.